### ISSUE TITLE
The file /etc/kbd/config doesn't exdist anymore

### ISSUE THREAD

**Issue by steev** [Re4son/RPi-Tweaks#8]
https://github.com/Re4son/RPi-Tweaks/blob/e0b15714bc41e0312f0eae053ef4b74033765825/kalipi-tft-config/kalipi-tft-config#L96 does a sed on /etc/kbd/config but that doesn't work as of anything newer than debian 9.3 (Kali rolling release is based on testing which is 10+) - we should actually be adding a `consoleblank=X` in `/boot/cmdline.txt` instead.

 (from hyperpixel repository README)
```
If you're on Raspbian Stretch (Debian 9.3), that config file doesn't exist. Instead, append consoleblank=0 to the boot command in /boot/cmdline.txt and reboot.```

**Comment by Re4son** [comment]
Good point. We've been creating the file [here](https://github.com/Re4son/RPi-Tweaks/blob/e0b15714bc41e0312f0eae053ef4b74033765825/kalipi-tft-config/kalipi-tft-config#L360)  for legacy reasons but might probably be about time to test if the cmdline option is supported by all the screens.
Leave that with me I'll do some tests
